code/modules/atmospherics/machinery/components/components_base.dm

```diff
@@ -100,7 +101,7 @@
 	return ..()
 
 /obj/machinery/atmospherics/components/on_construction()
-	..()
+	. = ..()
```

Comment:
does this change behavior at all? or is it just a consistency thing

Reply:
i changed it to test if it was this that wasn't making a thing work, but then i left it for consistency